### ISSUE TITLE
check for nonASCII entries

### ISSUE THREAD

**Issue by nicholasjhorton** [c1araaa/ElevatorsNYC#3]
The `tools::showNonASCII()` function will look at a string vector and let you know if there are non-ASCII characters.  

The `iconv()` function can help you remove these once you find them.

Please let me know if you run into any problems:

**Comment by c1araaa** [comment]
Hi Nick, I went through all the variables in the dataset (https://github.com/Amherst-STAT495F20/hw6-ind-c1araaa/commit/bfab7054a396794617902a2fab47beab0a1c53d7) using tools::showNonASCII() and could not find the error. I saved the result from running the showNonASCII() as a vector and compared the length of the vector to the number of NA's I had. The vector length and the number of NA's always matched up, which led me to believe that there is no nonASCII input. I'm not sure how to proceed and would love your suggestion!

**Comment by nicholasjhorton** [comment]
I found these in the `data-raw` csv file:

```
> tools::showNonASCIIfile("data-raw/elevators.csv")
10195: 1P16392,A,ACTIVE,1040140,1363,26,448,E 52 ST,100220000,Manhattan,Passenger Elevator (P),20150414,DF,,,,,2000,,,BG,<c2><ac>,D,19880711,C,16,,40.75404844811075,-73.9632781851441
10196: 1P16393,A,ACTIVE,1040140,1363,26,448,E 52 ST,100220000,Manhattan,Passenger Elevator (P),20150414,NV,,,,,2000,,,BG,<c2><ac>,A,19880711,B,ST,,40.75404844811075,-73.9632781851441
16210: 1P23279,A,ACTIVE,1036625,1310,1462,667,LEXINGTON AVE,100220000,Manhattan,Passenger Elevator (P),20150317,DF,,,,,2000,,,OG,<c2><ac>,A,19880711,C,17,,40.75991576410064,-73.9694258730646
16212: 1P23280,A,ACTIVE,1036625,1310,1462,667,LEXINGTON AVE,100220000,Manhattan,Passenger Elevator (P),20150317,DF,,,,,2000,,,OG,<c2><ac>,A,19880711,C,17,,40.75991576410064,-73.9694258730646
16213: 1P23281,A,ACTIVE,1036625,1310,1462,667,LEXINGTON AVE,100220000,Manhattan,Passenger Elevator (P),20150317,DF,,,,,2000,,,OG,<c2><ac>,A,19880711,C,17,,40.75991576410064,-73.9694258730646
18371: 1P2545,A,ACTIVE,1007408,492,38,192,BOWERY,100120000,Manhattan,Passenger Elevator (P),20150316,NV,,,,,1000,,,OH,,A,19880711,B,S<c2><a6>,,40.72119397245898,-73.99407271502349
24907: 1P32159,A,ACTIVE,1062836,2119,36,2020,AMSTERDAM AV,100320000,Manhattan,Passenger Elevator (P),20131115,DF,19830317,,,,2000,,,OG,I,A,19880711,L,<c2><ac>,,40.8352996213571,-73.94092904749449
29612: 1P3809,A,ACTIVE,1014260,778,33,211,W 28 ST,100010000,Manhattan,Passenger Elevator (P),20120625,DF,,,,,2000,,,OG,,A,19880711,B,S<c2><a6>,,40.74770004942525,-73.99389351404783
40807: 1P749,A,ACTIVE,1001026,47,7501,108,BROADWAY,100050000,Manhattan,Passenger Elevator (P),20150128,NV,,,,,2500,,,OG,,D,19880711,B,S<c2><a6>,,40.70821915870969,-74.01052131767499
50690: 2P5787,A,ACTIVE,2086851,3330,1,3020,WEBSTER AVENUE,0,Bronx,Passenger Elevator (P),20140915,DF,19960926,,,,1000,,,O,,H,19980515,B,S<c2><a6>,,40.869444589536094,-73.87878195614086
53242: 3P1041,A,ACTIVE,3178659,6681,415,1802,OCEAN PKWY,112230000,Brooklyn,Passenger Elevator (P),20131014,DF,19970718,A J ROSELL,,=A<c2><a2>,1500,,,BG,B,A,19970804,2,,,40.604462355100175,-73.96755236517612
64787: 3P8537,A,ACTIVE,3221964,7916,1,5811,DITMAS AVE,112030000,Brooklyn,Passenger Elevator (P),20131202,NV,20000324,,,,6400,,,O,I,E,20000404,B,<c2><a2>,,40.644080341166756,-73.92140198447035
```

**Comment by nicholasjhorton** [comment]
Note that the non-ascii characters show up as <c2><ac>:

https://www.ascii-code.com indicates this as Latin capital A with circumflex and not sign.  

They all seem to be in the same field.  

I would suggest deleting these using the `iconv()` command we discussed yesterday.

**Comment by c1araaa** [comment]
Hi Nick! I used `iconv()` and dplyr::na_if() to get rid of the non-ASCII characters (https://github.com/c1araaa/ElevatorsNYC/commit/6fb447d3fc803f6b04b17aee2daf28dd5c3349da). I double checked the dataset (ElevatorsNYC.rda in "data" folder) to confirm that I have completed the task. However, when I run the check under the "build" tab, the following error still appears. Am I missing a step? Have I not actually updated the dataset? 

> checking data for non-ASCII characters ... WARNING
    Warning: found non-ASCII string
    '=A<c2><a2>' in object 'ElevatorsNYC'
